### PR TITLE
Adds annotation for content-type of ASP.NET Web API endpoints

### PR DESCRIPTION
**Pull request by ZadenRB**
**Changes**
Adds a `Produces` annotation to the base controller so that all endpoints are marked as returning `application/json`

**Issues**
Part of #2872

### CODE REVIEW

File: Jellyfin.Api/BaseJellyfinApiController.cs [hunk 1]
```diff
@@ -7,6 +7,7 @@ namespace Jellyfin.Api
     /// </summary>
     [ApiController]
     [Route("[controller]")]
+    [Produces("application/json")]
```
Comment (cvium) [inline Jellyfin.Api/BaseJellyfinApiController.cs:10]: `MediaTypeNames.Application.Json`

https://docs.microsoft.com/en-us/dotnet/api/system.net.mime.mediatypenames.application.json?view=netcore-3.1

File: Jellyfin.Api/BaseJellyfinApiController.cs [hunk 2]
```diff
@@ -1,3 +1,4 @@
+using System.Net.Mime;
```
Comment (cvium) [inline Jellyfin.Api/BaseJellyfinApiController.cs:1]: @Bond-009 What is the order of imports?

Comment (ZadenRB) [reply Jellyfin.Api/BaseJellyfinApiController.cs:1]: I'm fairly confident System imports first is correct. When they are switched, Style Cop Analyzer throws an error.

**Comment by mark-monteiro** [comment]
I think this could go straight to master instead of the feature branch right?

File: Jellyfin.Api/BaseJellyfinApiController.cs (hunk 2 above)
Comment (Bond-009) [reply Jellyfin.Api/BaseJellyfinApiController.cs:1]: Correct, `System`first and after that alphabetical order

**Pull request merged**